apricot/local/service.go

```diff
@@ -135,12 +135,18 @@ func (s *Service) GetHostInventory(detector string) (hosts []string, err error)
 		log.WithError(err).Fatal("Error, could not retrieve host list.")
 		return []string{}, err
 	}
-	i := 0
-	hosts = make([]string, len(keys))
+	hostSet := make(map[string]bool, len(keys))
```

Comment:
I advise starting from zero length here, because `len(keys)` is only an upper bound here.